master/buildbot/test/fake/docker.py

```diff
@@ -41,8 +45,9 @@ def build(self, fileobj, tag):
             self._images.append({'RepoTags': [tag + ':latest']})
 
     def create_host_config(self, *args, **kwargs):
-        pass
+        self.call_args_create_host_config.append(kwargs)
 
-    def create_container(self, image, *args, **kwargs):
+    def create_container(self, *args, **kwargs):
```

Comment:
this is a change of method signature (`image` is no more welcome): is it OK?  is there a possibility that somebody uses this parameter for something outside of the Buildbot code base?